internal/infrastructure/event/factory.go

```diff
@@ -22,13 +22,13 @@ func Start(ctx context.Context, config *config.Kafka, handler Eventable) {
 	)
 
 	if consumer, err = NewConsumer(config); err != nil {
-		log.Panic().Msgf("error creating consumer: %s", err.Error())
+		slog.Error("error creating consumer", err.Error())
```

Comment:
`!BADKEY`

Reply:
Fixed

Reply:
Is it fixed? I still see the wrong call in the PR.